packages/onboarding/src/utils/flows.ts

```diff
@@ -77,9 +76,7 @@ export const isAnyHostingFlow = ( flowName?: string | null ) => {
 };
 
 export const isAnyMigrationFlow = ( flowName?: string | null ) => {
-	return Boolean(
-		flowName && [ IMPORT_FOCUSED_FLOW, IMPORT_HOSTED_SITE_FLOW ].includes( flowName )
-	);
+	return Boolean( flowName && [ IMPORT_FOCUSED_FLOW ].includes( flowName ) );
```

Comment:
Nit: We could simplify:

```suggestion
	return Boolean( flowName && IMPORT_FOCUSED_FLOW === flowName );
```

Reply:
I thought about that, then I also saw that the `array.includes` approach is used in most flows, even with single-item arrays—and so I opted for consistency. But also happy to be more YAGNI and switch to your suggestion

Reply:
It's a minor detail, I guess folks leave it with `Array.prototype.includes()` just to accommodate the addition of future steps/flows there. I prefer simplicity and YAGNI - we most likely will never add new flows/steps there. Up to you really.